### PR TITLE
N51: Untitled Pixel Wizards Game

### PR DESCRIPTION
**Pull request by caspark**
My first time sharing updates here for the game I've been working on for the last few months - please let me know if I messed anything up!

PS. super glad that this newsletter is rebooting!

_Part of #1505_

### CODE REVIEW

**Comment by janhohenheim** [comment]
Looks like you just made the deadline for the month! Will check out later :)

File: content/news/051/index.md [hunk 1]
```diff
@@ -90,6 +90,35 @@ You can support development by [purchasing Way of Rhea on Steam][wor], or
 [wor-mail]: https://anthropicstudios.com/newsletter/signup/tech
 [wor-cps]: https://www.cerebralpuzzleshowcase.com/
 
+### [Untitled Pixel Wizard Game][pixel-wizards]
+
+[![A target being destroyed by a rudimentary flame thrower spell](untitled-pixel-wizards-game.gif)][pixel-wizards]
```
Comment (Vrixyz) [inline content/news/051/index.md:95]: Looks like you forgot to add the gif ; for reminder, try to keep its size low (See [contributing.md](https://github.com/rust-gamedev/rust-gamedev.github.io/blob/source/CONTRIBUTING.md#style-guidelines))

Comment (caspark) [reply content/news/051/index.md:95]: :facepalm: woops - fixed! Thanks.

**Review by janhohenheim — APPROVED** [review]

**Comment by janhohenheim** [comment]
LGTM. Thanks for linking your blog posts as well! Just read through _Architectural Questions_, nice read :)
I'll maybe move them into a section lower in the newsletter so that people who skip the games section can still find them.

**Comment by caspark** [comment]
Sounds good to me - thanks for merging!

**Pull request merged**